app/controllers/feedback_controller.rb

```diff
@@ -0,0 +1,15 @@
+class FeedbackController < ApplicationController
+  def index
+    @skip_footer = true
+  end
+
+  def create
+    fields = [:bug, :context, :expected, :actually, :comments, :satisfaction, :referral]
+    fields = fields.select {|key| !params[key].blank?}
+    form_values = fields.map {|key| [key, params[key]]}
```

Comment:
Not sure that it makes a *huge* difference, but this is basically what [strong parameters](http://edgeguides.rubyonrails.org/action_controller_overview.html#strong-parameters) are for.